Fluid.MvcViewEngine/FluidRendering.cs

```diff
@@ -62,6 +62,9 @@ static FluidRendering()
             context.AmbientValues[ViewPath] = path;
             context.AmbientValues.Add("Sections", new Dictionary<string, IList<Statement>>());
 
+            var viewsPath = Path.Combine(_hostingEnvironment.ContentRootPath, "Views");
```

Comment:
I already did something similar to the above line, but I see myself make the partials folder as magic string which doesn't make sense to me

Let me show you my last attempt, after that you can review it if you don't mind